atomic_reactor/util.py

```diff
@@ -985,7 +985,95 @@ def get_manifest_list(image, registry, insecure=False, dockercfg_path=None):
     return response
 
 
-def get_config_from_registry(image, registry, digest, insecure=False,
+def get_all_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
+                             versions=('v1', 'v2', 'v2_list')):
+    """Return manifest digests for image.
+
+    :param image: ImageName, the remote image to inspect
+    :param registry: str, URI for registry, if URI schema is not provided,
+                          https:// will be used
+    :param insecure: bool, when True registry's cert is not verified
+    :param dockercfg_path: str, dirname of .dockercfg location
```

Comment:
Could add a description of `versions` here.

Reply:
will do